program/src/lib.rs

```diff
@@ -12,3 +17,21 @@ pub mod state;
 pub use solana_program;
 
 solana_program::declare_id!("ide3Y2TubNMLLhiG1kDL6to4a8SjxD18YWCYC5BZqNV");
+
+/// Given a DID, validate that the signers contain at least one
+/// account that has permissions to sign transactions using the DID.
+pub fn validate_owner(did: &AccountInfo, signers: &[AccountInfo]) -> ProgramResult {
+    if did.owner.ne(&id()) {
+        return Err(SolidError::IncorrectProgram.into());
```

Comment:
nit: you could use `ProgramError::IncorrectProgramId` instead, but that's totally up to you

Reply:
Ah cool-Done